articles/flows/guides/device-auth/includes/request-device-code.md

```diff
@@ -0,0 +1,70 @@
+## Request Device Code
+
+Once the user has started their device app and wants to authorize the device, you'll need to get a device code. When the user begins their session in their browser-based device, this code will be bound to that session. 
+
+To get the device code, your app must send the user to the [device code URL](/api/authentication#device-auth), including the Client ID.
+
+### Example POST to device code URL
+
+```har
+{
+  "method": "POST",
+  "url": "https://${account.namespace}/oauth/device/code",
+  "headers": [
+    { "name": "Content-Type", "value": "application/x-www-form-urlencoded" }
+  ],
+  "postData": { 
+    "text" : "{\"client_id\":\"${account.clientId}\",\"scope\":\"SCOPE\",\"audience\":\"AUDIENCE\" }"
+  }
+}
+```
+
+#### Parameters
+
+Note that when requesting a device code, you:
+
+- must include an audience parameter
```

Comment:
It depends if the app wants an access token for an api or if the app simply wants an access token to get information about the logged user (oidc).
Api => must
Profile => not needed

Reply:
Marking as unresolved, I still can see it

Reply:
it's below

```
::: note
 If your app wants an Access Token only to retrieve info about the authenticated user, then no audience parameter is required.
:::
```

Reply:
Technically, this entire guide is about calling a custom API using the flow (which is why configuring the API is a prerequisite) and focuses on getting an Access Token, as that's how we've written the other tutorials we have for other flows. For the moment, I have changed the intro sentence to "to call a custom API".

We could expand this, of course, but we'd need to add more detail about including openid and the /userinfo endpoint, etc. Do you suggest we switch the aim of this particular job-to-be-done? (And if so, do you think it's something we should do for this release, or iterate on?)

Reply:
Sure 👍 we are good.